src/Chapter09/Listing09.12.RecordConstructors.cs

```diff
@@ -0,0 +1,14 @@
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter09.Listing09_12;
+
+public record struct Angle(
+int Degrees, int Minutes, int Seconds, string? Name = null)
+{
+
+    #region INCLUDE
+    public Angle(
+        string degrees, string minutes, string seconds)
+        : this(int.Parse(degrees),
+              int.Parse(minutes), int.Parse(seconds))
```

Comment:
nit: I realize this is how it was in the book, but I think stacking these calls makes the example cleaner. 
```suggestion
               int.Parse(minutes), 
               int.Parse(seconds))
```